src/main/resources/config/reflectionconfig-javafx.json

```diff
@@ -908,4 +913,9 @@
     "name":"com.sun.javafx.logging.jfr.JFRPulseLogger",
     "methods":[{"name":"createInstance","parameterTypes":[] }]
   }
+,
+  {
+    "name":"com.sun.security.sasl.gsskerb.JdkSASL",
```

Comment:
why is this in JavaFX?

Reply:
actually it is duplicated from the java file